frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -67,6 +70,7 @@ type DashboardAppState = {
   mapDispatchToProps,
 )
 @title(({ dashboard }) => dashboard && dashboard.name)
+@loadingTimeTitle("loadingStartTime")
```

Comment:
It might be kind of nice if it were called `@titleWithLoadingTime` for consistency with `@title`?

```
@title(({ dashboard }) => dashboard && dashboard.name)
@titleWithLoadingTime("loadingStartTime")
```